app.js

```diff
@@ -0,0 +1,91 @@
+/* jshint esversion:6 */
+const express = require('express');
+const bodyParser = require('body-parser');
+const app = express();
+
+const port = 8080;
+
+let wordArr = [];
+let score = 0;
+
+let getBuzzArr;
+
+function handlePost( req, res ) {
+  let number = parseInt(req.body.points);
+  req.body.points = number;
+  wordArr.push(req.body);
+
+  validation(req, res);
+  console.log(wordArr);
+}
+
+function handleGet( req, res ) {
+  getBuzzArr = wordArr.map( obj => {
+    return obj.buzzWord;
+  });
+
+  let buzzObj = {
+    buzzWords: getBuzzArr
+  };
+
+  res.send(buzzObj);
```

Comment:
An alternative way to send this response (not better, just different)
`res.send({ buzzWords: getBuzzArr });`